draft-ietf-rtcweb-jsep.xml

```diff
@@ -1140,7 +1140,18 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                 </section>
 
                 <section title="addTrack" anchor="sec.addTrack">
-                  <t>[TODO]</t>
+                  <t>
+                    The addTrack method adds a MediaStreamTrack to the PeerConnection,
+                    using the MediaStream argument to associate the track with other
+                    tracks in the same MediaStream, so that they can be added to the
+                    same "LS" group when creating an offer or answer.
+                    addTrack attempts to minimize the number of transceivers as follows:
+                    The track will be attached to the first compatible transceiver
+                    (of the same media type) which has never had a direction of
+                    "sendonly" or "sendrecv". If no such transceiver exists,
```

Comment:
I think we need to note as an open issue that the reuse of 'recvonly'/'inactive' m= lines may not be correct.